src/snowflake/cli/_plugins/helpers/commands.py

```diff
@@ -35,10 +36,24 @@ def v1_to_v2(
     accept_templates: bool = typer.Option(
         False, "-t", "--accept-templates", help="Allows the migration of templates."
     ),
+    migrate_local_yml: bool = typer.Option(
+        False,
```

Comment:
As discussed offline, would prefer to make this abort the migration if a snowflake.local.yml is present, and make the user choose the behaviour explicitly.

Reply:
updated